packages/marketplace/contracts/royalties-registry/RoyaltiesRegistry.sol

```diff
@@ -239,7 +230,6 @@ contract RoyaltiesRegistry is IRoyaltiesProvider, OwnableUpgradeable {
                         }
                         // sum can be less than amount, otherwise small-value listings can break
                         require(sum <= royaltyAmount, "RoyaltiesRegistry: Invalid split");
-                        return royalties;
```

Comment:
I don't understand this removal, without it, it wil return an empty LibPart.Part[]

Reply:
Well, the royalties variable is a named return, so we don't need to use the return statement here. 
Anyway, I put it back here -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/1228/commits/7885eec84603204ba0def389f0f3218943abf25c